src/components/Catalog.vue

```diff
@@ -318,7 +318,7 @@ export default {
       legend_position:  ApplicationState.project.state.legend_position || 'tab',
       iframe:           ApplicationState.iframe,
       showlegend:       false,
-      backgroundLegend: ApplicationState.gui.layout.app.legend.transparent ? 'transparent' : '#FFFFFF', //@since 3.11.3 set transparent or white background
+      backgroundLegend: ApplicationState.gui.layout.app.legend && ApplicationState.gui.layout.app.legend.transparent ? 'transparent' : '#FFFFFF', //@since 3.11.3 set transparent or white background
```

Comment:
```suggestion
      backgroundLegend: ApplicationState.gui.layout.app?.legend?.transparent ? 'transparent' : '#FFFFFF', //@since 3.11.3 set transparent or white background
```